app/src/main/java/com/peacecorps/pcsa/policies_glossary/PoliciesFragment.java

```diff
@@ -14,14 +14,24 @@
 import com.peacecorps.pcsa.SingleTextViewFragment;
 
 /*
- * Policies and Glossary, Main Screen
+ * policies and glossary, main screen
+ * shows some extra relevant information related to peace corps and women safety
```

Comment:
This is similar as the previous one. The app is not only about women safety and this comment is unnecessary.